app/models/delivery_zone.rb

```diff
@@ -0,0 +1,14 @@
+class DeliveryZone < ActiveRecord::Base
+  validates :zipcode, presence: true, uniqueness: { case_sensitive: false }
```

Comment:
OK, that makes sense. Thoughts on validating that it's only numerical data as well?